presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -0,0 +1,137 @@
+=======================
+Elasticsearch Connector
+=======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+The Elasticsearch Connector for Presto allows access to Elasticsearch data from Presto. This tutorial shows how to set up the Elasticsearch Connector to run SQL on Elasticsearch.
+
+.. note::
+
+    It is highly recommend to use Elasticsearch 5.1.2 or later.
+
+Configuration
+-------------
+
+To configure the Elasticsearch connector, create a catalog properties file
+``etc/catalog/elasticsearch.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=elasticsearch
+    elasticsearch.table-names=schema.table1,schema.table2
+    elasticsearch.default-schema=default
+    elasticsearch.table-description-dir=etc/elasticsearch/
+    elasticsearch.scroll-size=1000
+    elasticsearch.scroll-time=60000
+
+
+Multiple Elasticsearch Clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can have as many catalogs as you need, so if you have additional
+Elasticsearch clusters, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``). For
+example, if you name the property file ``users.properties``, Presto
+will create a catalog named ``users`` using the configured connector.
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+======================================= ==============================================================================
+Property Name                           Description
+======================================= ==============================================================================
+``elasticsearch.table-names``           List of all tables provided by the catalog
+``elasticsearch.default-schema``        Default schema name for tables
+``elasticsearch.table-description-dir`` Directory containing topic description files
+``elasticsearch.scroll-size``           Maximum number of hits to be returned with each batch of Elasticsearch scroll
+``elasticsearch.scroll-time``           Number of milliseconds the Elasticsearch would keep live for a search context
```

Comment:
Can you rephrase this sentence? It's not clear what Elasticsearch will keep live for this amount of time. Also remove `the` before `Elasticsearch`.